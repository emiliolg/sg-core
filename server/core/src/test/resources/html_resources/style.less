.some {
  color: red;

  .inner {
    color: blue;
  }
}
