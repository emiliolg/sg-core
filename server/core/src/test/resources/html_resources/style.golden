.some{color:#f00}.some .inner{color:#00f}
